README.md

```diff
@@ -16,3 +16,11 @@ Then bot will be available in slack.
 Deployment is run automatically after merging changes to the master branch.
 
 Mona deployed on Heroku.
+
+## Testing
+
```

Comment:
Thoughts about describing which modules we use for testing?

Reply:
going to add these notes